src/Spectre.libClassifier/Svm.h

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
Sometimes I wonder, what does this git find different in these lines... @mg6? I checked it twice yesterday - there's even no whitespace difference...

Reply:
Actually, file encoding has changed (BOM removed), so there was a whitespace difference.
```diff
% git show edcdabc -- src/Spectre.libClassifier/Svm.h
commit edcdabcaef5ee3b86ad2e8ddb6c6927d89e957fd
Author: Regis128 <regis1286@gmail.com>
Date:   Thu Jan 4 21:39:21 2018 +0100

    Fixed indents in Svm

diff --git a/src/Spectre.libClassifier/Svm.h b/src/Spectre.libClassifier/Svm.h
index a990b80..2499492 100644
--- a/src/Spectre.libClassifier/Svm.h
+++ b/src/Spectre.libClassifier/Svm.h
@@ -1,4 +1,4 @@
-<U+FEFF>/*
+/*
 * Svm.h
 * Simple wrapper for OpenCV SVM.
 *
(redacted)
```